src/Consumer.cc

```diff
@@ -133,10 +135,19 @@ class ConsumerNewInstanceWorker : public Napi::AsyncWorker {
     }
 
     this->done = false;
-    if (topic.empty()) {
+    if (!topicsPattern.empty()) {
       pulsar_client_subscribe_pattern_async(this->cClient, topicsPattern.c_str(), subscription.c_str(),
                                             this->consumerConfig->GetCConsumerConfig(),
                                             &ConsumerNewInstanceWorker::subscribeCallback, (void *)this);
+    } else if (topics.size() > 0) {
+      const char **c_topics = new const char *[topics.size()];
```

Comment:
Is there any reason why this variable is not in camel case?

Reply:
@massakam 
Thank you for your time.
I apologize for the snake_case. It was a force of habit.

I've updated the code from snake_case to camelCase (```c_topics``` → ```cTopics```).
Please let me know if you would like me to make any further changes.